### PR TITLE
Classes

### PR DESCRIPTION
**Pull request by ryan-novak**
## What does this PR do?
This PR adds the creation of classes that will be used. Also begins login check for the manager, and will begin running functionality based on what the manager has access to.

## Why are we doing this? Any context or related work?
Beginning stages of functionality for this project.

## Where should a reviewer start?
Changes are made in index.js, DomUpdate.js, and hotel.js files.

@vladd-png

### CODE REVIEW

File: src/DomUpdate.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        `
+    );
+  }
+
+  showLoginError() {
+    $(".login-error-message").css("display", "block");
+  }
+
+  removeLoginError() {
+    $(".login-error-message").css("display", "none");
```
Comment (vladd-png) [inline src/DomUpdate.js]: Really cool jquery here, I didn't know this syntax.

File: src/DomUpdate.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  logout() {
+
+  }
+
+  loadUser() {
+
+  }
+
+  loadManager() {
+    $(".welcome-message").text("WELCOME BACK, NIGHTHAWK!");
```
Comment (vladd-png) [inline src/DomUpdate.js]: 👏🏻😂

File: src/css/base.scss [hunk 3]
```diff
@@ -1,6 +1,108 @@
...
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  text-align: center;
+  height: 60px;
+}
+
+.username-input,
```
Comment (vladd-png) [inline src/css/base.scss]: Could create an @extends here with Sass

**Comment by vladd-png** [comment]
Great PR start. Looks like you're not creating booking or room classes. Do you plan to fetch the data and sort it as is? Also your DOM updates file looks great, and I love the syntax you used with jQuery .css(value1, value2).

**Comment by ryan-novak** [comment]
> Great PR start. Looks like you're not creating booking or room classes. Do you plan to fetch the data and sort it as is? Also your DOM updates file looks great, and I love the syntax you used with jQuery .css(value1, value2).

Initially I had the booking and room classes, but wasn't too sure methods could really live within them.... I thought about doing the hotel class, which holds the data for rooms and data and feel like the methods could live within that class..... pretty sure it can be done both ways, still debating which would be easier to use

**Comment by vladd-png** [comment]
I did a 'hotel' class as well, to organize the rooms and bookings. I think that's a good idea

**Pull request merged**